composer.json

```diff
@@ -86,8 +86,9 @@
             },
             "drupal/url_embed": {
                 "WSOD if wrong url or network unavailable": "https://www.drupal.org/files/issues/url_embed_WSOD_convert_url_to_embed-2871744-5.patch",
-                "Improve performance for embeds": "https://www.drupal.org/files/issues/2018-03-16/url_embed-caching-layer-2867668-17.patch",
-                "Translate dialog title": "https://www.drupal.org/files/issues/2018-03-16/url_embed_translate_dialog_title-2953591-2.patch"
+                "Translate dialog title": "https://www.drupal.org/files/issues/2018-03-16/url_embed_translate_dialog_title-2953591-2.patch",
+                "Improve performance for embeds": "https://www.drupal.org/files/issues/2020-04-08/url_embed-n2867668-23.patch",
+                "Improve how the module deals with non-embeddable URLs #2761187 (See: https://www.drupal.org/project/social/issues/2930457#comment-13790581)": "https://www.drupal.org/files/issues/2020-08-18/urlembed-non-embeddable-urls-2761187-opensocial-combined-3.patch"
```

Comment:
This one should be improved, see my comment on the PR itself.

Reply:
The new patch is added here: https://github.com/goalgorilla/open_social/pull/1953/commits/1f23367c34cb99d695b7b0923a15867cd00a32cc